test/e2e/framework.go

```diff
@@ -100,8 +100,8 @@ func (f *Framework) WaitForPodRunning(podName string) error {
 }
 
 // Runs the given pod and verifies that its output matches the desired output.
-func (f *Framework) TestContainerOutput(scenarioName string, pod *api.Pod, expectedOutput []string) {
-	testContainerOutputInNamespace(scenarioName, f.Client, pod, expectedOutput, f.Namespace.Name)
+func (f *Framework) TestContainerOutput(scenarioName string, pod *api.Pod, containerId int, expectedOutput []string) {
+	testContainerOutputInNamespace(scenarioName, f.Client, pod, containerId, expectedOutput, f.Namespace.Name)
```

Comment:
Why do you pass through containerid instead of container name here.  The convention in the rest of the code seems to deal with container names.